data/messages.json

```diff
@@ -330,7 +330,8 @@
   "KILLTIME_MINIMUM_TWO_MINUTES": "{DC143C}Killtime{FFFFFF}: Sorry, there is a minimum of two minutes.",
   "KILLTIME_STARTED": "{DC143C}Killtime{FFFFFF}: The following %d minutes is killing for a prize! Are you the one with the most kills?",
   "KILLTIME_TOPKILLER": "{DC143C}Killtime{FFFFFF}: Leading: %s. Hurry up and get more to win the prize of $%d or more!",
-  "KILLTIME_USAGE": "{DC143C}Killtime{FFFFFF}: '/killtime start 4' for 4 minutes killtime, none = 2 minutes. '/killtime stop' to end.",
+  "KILLTIME_USAGE": "{DC143C}Killtime{FFFFFF}: '/killtime start 4 35' for 4 minutes killtime, none = 2 minutes. '/killtime stop' to end.",
```

Comment:
Since this is an example usage, perhaps also mention which weapon?

Reply:
https://wiki.sa-mp.com/wiki/SendClientMessage

^Allows 144 characters at max. Adding that info would exceed that limit.
Also I expect admins to be smart enough, there's the /weapons command after all.